cvpaperchallenge/Ascender
# 67/enh/introduce markdown formatter

## Issue URL

#67

## Change overview

- Add a new markdown formatter called [mdformat](https://github.com/executablebooks/mdformat)
- Add new phoney commands in Makefile for GitHub Actions

## How to test

Check the changed file and see whether the test of this pull request has been passed without any errors.

## Note for reviewers

As Prettier requires to install Node.js and causes unintended complexity, I introduced another open-source markdown formatter called [mdformat](https://github.com/executablebooks/mdformat), which is provided as a python library.  The rules that mdformat follows are summarized [here](https://mdformat.readthedocs.io/en/stable/users/style.html).  Please leave comments if you notice potential problems due to those formatting rules.

close #67
